src/api/goods.ts

```diff
@@ -6,6 +6,18 @@ export function getAll(): Promise<Good[]> {
     .then(response => response.json());
 }
 
-export const get5First = () => {};
+export const get5First = () => {
+  return fetch(API_URL)
+    .then(response => response.json())
+    .then(goods => {
```

Comment:
Here you have doubled the getAll () function: 
```suggestion
export const get5First = () => {
  return getAll()
    .then(goods => {
```